app/main/posts/views/post-view-data.directive.js

```diff
@@ -173,6 +255,26 @@ function PostViewDataController(
         getPosts();
     }
 
+    function selectBulkActions() {
+        $scope.bulkActionsSelected = 'toolbar-active';
+        $rootScope.$broadcast('bulkActionsSelected:true');
+    }
+
+    function closeBulkActions() {
+        $scope.bulkActionsSelected = '';
+        $rootScope.$broadcast('bulkActionsSelected:false');
```

Comment:
This broadcast does not seem to be picked up anywhere = checkboxes still visible after closing bulk-actions